woocommerce/woocommerce-android
# Issue/5712 Specifc error message when not supported reader is used

Closes: #5712


### Description

* Shows a toast message with an error message from the SDK
* Removes Canada IPP flag check on what type of readers we want to scan for 

### Testing instructions

* Try to collect payment with none Wisepad 3 readers in Canada store
* Notice a toast message

### Images/gif


https://user-images.githubusercontent.com/4923871/153865353-d6ac7823-2cf7-4e44-b9e1-55d3b6be76c3.mp4
